lib/router.dart

```diff
@@ -226,8 +226,7 @@ Route<dynamic> generateRoute(RouteSettings settings) {
       );
     case Routes.pinnedPostScreen:
       // final Map<String, dynamic> post = settings.arguments! as Map<String, dynamic> ;
-      final Map<String, String> post =
-          settings.arguments! as Map<String, String>;
+      final Post post = settings.arguments! as Post;
```

Comment:
why add this?

Reply:
Sir, In the pinnedpost widget if you see the constructor you can see a variable pinnedPost which is a list of post but it was used nowhere instead a manually made list of map string was used so I made that widget to use pinnedPost. But the pinnedpostscreen widget was requiring a map string so I made a change there a made it to use post

Reply:
@Ayush0Chaudhary Please have a look.